src/NServiceBus.AcceptanceTests/Core/FakeTransport/CriticalError/When_registering_custom_critical_error_handler.cs

```diff
@@ -7,24 +7,20 @@
     using Configuration.AdvanceExtensibility;
     using EndpointTemplates;
     using NUnit.Framework;
-    using ScenarioDescriptors;
 
     public class When_registering_custom_critical_error_handler : NServiceBusAcceptanceTest
     {
         [Test]
-        public Task Critical_error_should_be_raised_inside_delegate()
+        public async Task Critical_error_should_be_raised_inside_delegate()
         {
-            return Scenario.Define<Context>()
+            var context = await Scenario.Define<Context>()
                 .WithEndpoint<EndpointWithLocalCallback>(b => b.When(
-                    (session, context) => session.SendLocal(new MyRequest())))
+                    session => session.SendLocal(new MyRequest())))
                 .Done(c => c.ExceptionReceived)
-                .Repeat(r => r.For(Transports.AllAvailable.SingleOrDefault(t => t.Key == "FakeTransport")))
```

Comment:
yeah, fair point but since there was just this usage, I decided to just sneak that past.